run_tests.py

```diff
@@ -78,6 +78,10 @@ def main():
             run_test("cpp", "lpython --no-color --show-cpp {infile}",
                     filename, update_reference, extra_args)
 
+        if tokens:
+            run_test("cpp", "lpython --no-color --show-tokens {infile} -o {outfile}",
```

Comment:
I think you forgot to update the prefix:
```suggestion
            run_test("tokens", "lpython --no-color --show-tokens {infile} -o {outfile}",
```

Reply:
(You'll have to update the tests --- remove everything in `tests/reference/*` and rerun `./run_tests.py -u`.

Reply:
Good catch, Thanks!